opengemini-client-api/pom.xml

```diff
@@ -11,4 +11,10 @@
 
     <artifactId>opengemini-client-api</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Can we remove this dependency from `opengemini-client-api`

Reply:
Annotation @JsonIgnoreProperties is used in class SeriesResult